### ISSUE TITLE
mkdir for post-backup scripts not possible

### ISSUE THREAD

**Issue by joergmschulz** [tiredofit/docker-db-backup#279]
While I don't have any post- or pre- backup scripts yet, I see an error message during backup:
```
/assets/functions/10-db-backup: line 2068: print_warning: command not found
mkdir: cannot create directory ‘/assets/scripts’: Permission denied
```

**Comment by tiredofit** [comment]
Thanks! I see the problem. I should be using `warn` instead of `warning` which will fix the command not found, and I'm also running as `dbbackup` when creating a folder where I should be doing it as root.

This is resolved in `4.0.14`

**Comment by joergmschulz** [comment]
the 'cannot create directory' error is gone .
The warning message persists:
 [01-postgres-db__ALL] Sending Backup Statistics to Zabbix
/assets/functions/10-db-backup: line 2068: print_warning: command not found. Changing lines 1634, 1703 to 'warn' does the job
